yarn/metadata.csv

```diff
@@ -22,15 +22,24 @@ yarn.metrics.lost_nodes,gauge,,node,,The number of lost nodes,0,yarn,nds lst
 yarn.metrics.unhealthy_nodes,gauge,,node,,The number of unhealthy nodes,0,yarn,nds unhlthy
 yarn.metrics.decommissioned_nodes,gauge,,node,,The number of decommissioned nodes,0,yarn,nds decomm
 yarn.metrics.rebooted_nodes,gauge,,node,,The number of rebooted nodes,0,yarn,nds rbtd
-yarn.apps.progress,rate,,percent,,The progress of the application as a percent,0,yarn,app prog
-yarn.apps.started_time,rate,,second,,The time in which application started (in ms since epoch),0,yarn,app strt tim
-yarn.apps.finished_time,rate,,second,,The time in which the application finished (in ms since epoch),0,yarn,app fin tim
-yarn.apps.elapsed_time,rate,,second,,The elapsed time since the application started (in ms),0,yarn,app elapsd tm
-yarn.apps.allocated_mb,rate,,mebibyte,,The sum of memory in MB allocated to the applications running containers,0,yarn,app mem alloc
-yarn.apps.allocated_vcores,rate,,core,,The sum of virtual cores allocated to the applications running containers,0,yarn,app cros alloc
-yarn.apps.running_containers,rate,,,,The number of containers currently running for the application,0,yarn,app ctrs rrun
-yarn.apps.memory_seconds,rate,,second,,The amount of memory the application has allocated (megabyte-seconds),0,yarn,app mem tm
-yarn.apps.vcore_seconds,rate,,second,,The amount of CPU resources the application has allocated (virtual core-seconds),0,yarn,app cor tm
+yarn.apps.progress_gauge,gauge,,percent,,The progress of the application as a percent,0,yarn,app prog
+yarn.apps.started_time_gauge,gauge,,second,,The time in which application started (in ms since epoch),0,yarn,app strt tim g
+yarn.apps.finished_time_gauge,gauge,,second,,The time in which the application finished (in ms since epoch),0,yarn,app fin tim g
+yarn.apps.elapsed_time_gauge,gauge,,second,,The elapsed time since the application started (in ms),0,yarn,app elapsd tm g
```

Comment:
```suggestion
yarn.apps.elapsed_time_gauge,gauge,,millisecond,,The elapsed time since the application started (in ms),0,yarn,app elapsd tm g
```

according to the desc should be millisecond, maybe need to be double checked.